lib/routes/home/widgets/payments_list/dialog/closed_channel_payment_details.dart

```diff
@@ -16,62 +18,63 @@ class ClosedChannelPaymentDetailsWidget extends StatelessWidget {
+                  text: TextSpan(
+                    style: themeData.dialogTheme.contentTextStyle,
+                    text: texts.payment_details_dialog_closed_channel_local_wallet,
+                  ),
+                ),
+                if (paymentMinutiae.paymentType == sdk.PaymentType.ClosedChannel &&
+                    paymentMinutiae.fundingTxid != null) ...[
+                  TxWidget(
+                    txURL: "$blockExplorer/tx/${paymentMinutiae.closingTxid!}",
+                    txID: paymentMinutiae.closingTxid!,
+                  ),
+                ],
+              ],
+            );
+          }
 
-    if (paymentMinutiae.status == PaymentStatus.Complete) {
-      return Column(
-        mainAxisSize: MainAxisSize.min,
-        children: [
-          RichText(
-            text: TextSpan(
-              style: themeData.dialogTheme.contentTextStyle,
-              text: texts.payment_details_dialog_closed_channel_local_wallet,
-            ),
-          ),
-          // TODO waiting for closeChannelTxUrl and closeChannelTx
-          const TxWidget(
-            txURL: "",
-            txID: "",
-          ),
-        ],
-      );
-    }
+          String estimation = texts.payment_details_dialog_closed_channel_transfer_no_estimation;
 
-    int lockHeight = 0; // TODO pendingExpirationHeight
-    double hoursToUnlock = 0.0; // TODO hoursToExpire
-
-    int roundedHoursToUnlock = hoursToUnlock.round();
-    String hoursToUnlockStr = roundedHoursToUnlock > 1
-        ? texts.payment_details_dialog_closed_channel_about_hours(
-            roundedHoursToUnlock.toString(),
-          )
-        : texts.payment_details_dialog_closed_channel_about_hour;
-    String estimation = lockHeight > 0 && hoursToUnlock > 0
-        ? texts.payment_details_dialog_closed_channel_transfer_estimation(
-            lockHeight,
-            hoursToUnlockStr,
-          )
-        : texts.payment_details_dialog_closed_channel_transfer_no_estimation;
-
-    return Column(
-      mainAxisSize: MainAxisSize.min,
-      children: [
-        RichText(
-          text: TextSpan(
-            style: themeData.dialogTheme.contentTextStyle,
-            text: estimation,
-          ),
-        ),
-        // TODO waiting for closeChannelTxUrl and closeChannelTx
-        const TxWidget(
-          txURL: "",
-          txID: "",
-        ),
-        // TODO waiting for remoteCloseChannelTxUrl and remoteCloseChannelTx
-        const TxWidget(
-          txURL: "",
-          txID: "",
-        ),
-      ],
+          return Column(
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              RichText(
+                text: TextSpan(
+                  style: themeData.dialogTheme.contentTextStyle,
+                  text: estimation,
+                ),
+              ),
+              TxWidget(
+                txURL: "$blockExplorer/tx/${paymentMinutiae.closingTxid!}",
+                txID: paymentMinutiae.fundingTxid!,
```

Comment:
Are we sure paymentMinutiae.fundingTxid is not null here?

Also the url & id do not match.

Reply:
Fixed